src/fsharp/SyntaxTree.fsi

```diff
@@ -808,6 +808,17 @@ type SynExpr =
         range: range *
         trivia: SynExprIfThenElseTrivia
 
+    /// F# syntax: if! expr then expr
+    /// F# syntax: if! expr then expr else expr
+    | IfBangThenElse of
+        ifExpr: SynExpr *
+        thenExpr: SynExpr *
+        elseExpr: SynExpr option *
+        spIfToThen: DebugPointAtBinding *
+        isFromErrorRecovery: bool *
+        range: range *
+        trivia: SynExprIfThenElseTrivia
```

Comment:
Please extend the `SynExprIfThenElseTrivia` to include `elif!` keyword where we today have `IsElif`